### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by doodi-v1** [ngrok/ngrok#13] [General]
Lately I have been sporadically receiving the following for a development site on my desktop using ngrok:

> This site can’t provide a secure connection [my-domain-here].ngrok-free.app sent an invalid response.
> Try running Windows Network Diagnostics.
> ERR_SSL_PROTOCOL_ERROR

I have used ngrok for months without issue. This is a new thing. I normally use chrome. For kicks, I tried MS Edge, it is experiencing the same. Anyone have any ideas?

**Comment by russorat** [comment]
Thanks for the issue! I'm following up internally to see if anyone has any ideas. 

Is there anything bespoke about your network configuration? Do you have any network antivirus or mitm proxies running?

**Reply by doodi-v1** [reply]
Subsequently I did find that windows, chrome, or google (one or all) had the ngrok client app flagged as dangerous due to malware possibly being able to use it. No malware was detected, just a dangerous security risk. Once I clicked that I understood the risks, no issues since.

**Comment by XJimmy098** [comment]


**Reply by russorat** [reply]
@XJimmy098 using ngrok with tools designed to phish information or spy on individuals is against our terms of service. https://ngrok.com/tos
